Index: SUSYTools/Root/SUSYCrossSection.cxx
===================================================================
--- SUSYTools/Root/SUSYCrossSection.cxx	(revision 758758)
+++ SUSYTools/Root/SUSYCrossSection.cxx	(working copy)
@@ -1,5 +1,4 @@
 #include "SUSYTools/SUSYCrossSection.h"
-#include "PathResolver/PathResolver.h"
 #include <dirent.h>
 #include <math.h>
 #include <iostream>
@@ -7,25 +6,6 @@
 using namespace std;
 
 SUSY::CrossSectionDB::CrossSectionDB(const std::string& txtfilenameOrDir, bool usePathResolver, bool isExtended) {
-  
-  setExtended(isExtended);
-  
-  if (usePathResolver) {
-    std::string fullPath = PathResolverFindCalibDirectory(txtfilenameOrDir);
-    
-    DIR* dp = opendir(fullPath.c_str());
-    if (dp) {
-      struct dirent * de;
-      while ((de = readdir(dp)) != NULL) {
-	loadFile(fullPath + de->d_name);
-      }
-    }
-    else {
-      std::string fullPathToFile = PathResolverFindCalibFile(txtfilenameOrDir);
-      loadFile(fullPathToFile.c_str());
-    }
-  }
-  else {
     DIR* dp = opendir(txtfilenameOrDir.c_str());
     if (dp) {
       struct dirent * de;
@@ -35,7 +15,6 @@
     } else {
       loadFile(txtfilenameOrDir.c_str());
     }
-  }
 }
 
 void SUSY::CrossSectionDB::loadFile(const std::string& txtfilename){
