Index: SUSYTools/Root/SUSYCrossSection.cxx
===================================================================
--- SUSYTools/Root/SUSYCrossSection.cxx	(revision 808522)
+++ SUSYTools/Root/SUSYCrossSection.cxx	(working copy)
@@ -1,31 +1,12 @@
 #include "SUSYTools/SUSYCrossSection.h"
-#include "PathResolver/PathResolver.h"
 #include <dirent.h>
 #include <math.h>
 #include <iostream>
 
 using namespace std;
 
-SUSY::CrossSectionDB::CrossSectionDB(const std::string& txtfilenameOrDir, bool usePathResolver, bool isExtended) {
+SUSY::CrossSectionDB::CrossSectionDB(const std::string& txtfilenameOrDir) {
   
-  setExtended(isExtended);
-  
-  if (usePathResolver) {
-    std::string fullPath = PathResolverFindCalibDirectory(txtfilenameOrDir);
-    
-    DIR* dp = opendir(fullPath.c_str());
-    if (dp) {
-      struct dirent * de;
-      while ((de = readdir(dp)) != NULL) {
-        loadFile(fullPath + de->d_name);
-      }
-    }
-    else {
-      std::string fullPathToFile = PathResolverFindCalibFile(txtfilenameOrDir);
-      loadFile(fullPathToFile.c_str());
-    }
-  }
-  else {
     DIR* dp = opendir(txtfilenameOrDir.c_str());
     if (dp) {
       struct dirent * de;
@@ -35,7 +16,7 @@
     } else {
       loadFile(txtfilenameOrDir.c_str());
     }
-  }
+
 }
 
 void SUSY::CrossSectionDB::loadFile(const std::string& txtfilename){
Index: SUSYTools/SUSYTools/SUSYCrossSection.h
===================================================================
--- SUSYTools/SUSYTools/SUSYCrossSection.h	(revision 808522)
+++ SUSYTools/SUSYTools/SUSYCrossSection.h	(working copy)
@@ -19,7 +19,7 @@
 class CrossSectionDB
 {
 public:
-  CrossSectionDB(const std::string& txtfilenameOrDir = "SUSYTools/data/mc15_13TeV/", bool usePathResolver = false, bool isExtended = false);
+  CrossSectionDB(const std::string& txtfilenameOrDir = "SUSYTools/data/mc15_13TeV/");
 
   // Load all the information from a file
   void loadFile(const std::string&);
